xml/Microsoft.VisualBasic.ApplicationServices/WindowsFormsApplicationBase.xml

```diff
@@ -1306,7 +1306,7 @@ Protected Overloads Shared ReadOnly Property UseCompatibleTextRendering() As Boo
 End Property  
 ```  
   
- You must enter this code in the Code Editor window for application events. For more information, see [Application Page, Project Designer (Visual Basic)](../Topic/Application%20Page,%20Project%20Designer%20\(Visual%20Basic\).md).  
+ You must enter this code in the Code Editor window for application events. For more information, see [Application Page, Project Designer (Visual Basic)](http://msdn.microsoft.com/library/8cec9fea-cd92-47ff-88dd-7c928f0b4a74).  
```

Comment:
The topic exists in docs: https://docs.microsoft.com/en-us/visualstudio/ide/reference/application-page-project-designer-visual-basic